geoalchemy2/shape.py

```diff
@@ -43,9 +48,18 @@ def from_shape(shape, srid=-1):
         An integer representing the spatial reference system. E.g. 4326.
         Default value is -1, which means no/unknown reference system.
 
+    ``use_ewkb``
+
+        A boolean to indicate whether to use EWKB or WKB.
```

Comment:
-> Boolean indicating whether converting the shape to a `WKBElement` or to a `EWKBElement`.